howso/distances.amlg

```diff
@@ -390,21 +431,34 @@
 			valid_weight_feature (false)
 		)
 
+		(declare (assoc
+			error_text
+				(if (and feature_values features (!= (size features) (size feature_values)))
+					"Specified features must be of same length as feature_values."
+
+					(and (!= (null) case_indices) (<= (size case_indices) 1) )
```

Comment:
indentation on this condition looks incorrect on the GH diff, check locally please

Reply:
GH issue.